static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -0,0 +1,171 @@
+@import 'svg';
+
+
+$contribute-yellow: #ffbb00;
+$gu-blue: #005689;
+$gu-blue-dark: #003d70;
+$comment-support-1: #7d7569;
+$white: #ffffff;
+$black: #000000;
+$gs-gutter: 20px;
+
+
+
+.interactive-embed {
+    height: 100%;
+    width: 100%;
+    border-top: 1px solid #4bc6df;
+    background-color: #efefec;
+
+    p {
+        line-height: 24px;
+        font-family: 'Guardian Egyptian Web';
+        padding-left: 5px;
+        padding-right: 15px;
+        margin-top: 8px;
+        margin-bottom: 8px;
+
+
+    }
+
+    .form-section {
+        font-family: 'Guardian Egyptian Web';
+        font-size: 1em;
+        line-height: 20px;
+        margin-top: 8px;
+        margin-bottom: 16px;
+    }
+
+
+
+    .title {
+        margin-top: 0;
+        margin-bottom: 2px;
+        padding-left: 5px;
+        padding-top: 7px;
+        font-family: 'Guardian Egyptian Web';
+        font-weight: bold;
+        font-size: 1.17em;
+    }
+
+
+    @mixin circular {
+        border-radius: 1000px; // Android 2.3 cannot deal with '50%'
+    }
+
+    @mixin button {
+        @include fs-textSans(4);
+        @include circular;
+
+        font-weight: bold;
+        text-decoration: none;
+        cursor: pointer;
+        text-align: left;
+        display: inline-block;
+        vertical-align: bottom;
+        outline: none;
+
+        padding: 6px $gs-gutter/2;
+        border: 0;
+        background-color: #ffffff;
+
+        &:hover {
+            background-color: $comment-support-1;
+            color: $white;
+        }
+
+        &.active {
+            background-color: $contribute-yellow;
+            color: $black;
+        }
+    }
+
+    .option-button {
+        @include button();
```

Comment:
There's not really any need to define this button as a mixin when you only use it here